README.dev-quickstart.md

```diff
@@ -0,0 +1,34 @@
+
+# hello world
```

Comment:
Dalsie README mi pride trochu zbytocne:
- popis co sa nachadza v ktorych adresaroch - to je super, moze sa doplnit do hlavneho README
- ako zacat a instalovat - [to tam uz mame](https://github.com/slovensko-digital/priznanie-digital#how-to-develop)
- ako pustat testy [tam je tiez](https://github.com/slovensko-digital/priznanie-digital#run-tests)
- [kde je to deploynute](https://github.com/slovensko-digital/priznanie-digital#environments)
- myslim ze hlavne README nie je take dlhe ze by potrebovalo `tl;dr` :)

Ak ide o to ze tam je to po anglicky, tak kludne to po dohode s ostatnimi developermi mozete prelozit. My sme to s @julius-retzer pisali po anglicky lebo sme tak vyknuti z prace (aspon teda ja).

Reply:
> Niektore subory vyzeraju ze su nove, ale pravdepodobne iba git nepochopil ze boli presunute. Na prvy pohlad sa mi neda ze tam boli nejake zasadne zmeny, ale mozno sa mylim.
> 
> V buducnosti by som radil robit taketo presuny cez [git mv](https://git-scm.com/docs/git-mv) aby sme mali lepsie diffy pri review.

menila sa len struktura, zmeny v suboroch boli len na urovni zmenenych paths, preto som sa aj nebabral s mv, ale inak suhlasim, ze ked sa robia zmeny, tak je dobre to robit na dve fazy, ze presuvanie zvlast, a zmeny v subore zvlast, pretoze nic nie je horsie ako vidiet stary a novy subor a nemoct spravit review zmien, 100% za

Reply:
Na uplny quick start mi pride ze staci sekcia `How to develop` kde je vlastne napisane ze `yarn && yarn start`.

Reply:
ja som zvyknuty ze som takto zastresoval 20 projektov naraz na 4 roznych stackoch, a zatial co readme je kompletny s celym info, tak pokial ide o rozbehanie projektu, tak som vzdy na projekte pozadoval od koderov mat quick guide, aby bol lahky onboarding novych ludi, ze nemusia hladat ihlu v kope sena - cize za mna to nie je tldr, ale 90% informacii nepotrebuje clovek citat, staci mu povedat ze ked sa s tym chces pohrat, tak staci ked si pozries quickstart - uz len informacia ci pouzivat yarn, alebo to ze po clone hned riesit yarn a nemusis nic robit, toto povazujem za klucove, a taktiez je to proof toho ze kod neni prekomplikovany lebo sa stava ze navody su dlhsie a projekt je pain, a toto je potom taky proof cistoty, a takisto je to super v tom, ze ak nejaky problem vyskoci, tak je tam quick faq a clovek nemusi dohladavat a vypytovat sa, ale rovno to rozbeha a bezne issues su tam v skratke zachytene a usmernene

ano, github otvori readme, to je dobre, a ked niekoho chces onboardnut tak mu povies ze rozbehaj si to, vid quickstart, nemusis nicomu inemu nerozumiet, druhy krok mu dam pozriet strukturu a vygenerovat otazky, atd (ale ako pises, vacsina ludi sa tam nedoklika, to neni ciel, ciel je mat pomocku na onboarding ludi)

naonboardoval som kopec ludi do desiatok projektov a viem co funguje, a pravdupovediac neviem opat teraz ci to co pises su ze nesuhlasy, alebo otazky ze co ma k tomu viedlo, lebo mi to neznie ako otazky, ale ja kludne budem diskutovat ci to dava zmysel, a nie vsetko dava, niektore veci mi jules ofeedbackoval a davalo mi to zmysel, takze osobne preferencie tlacit nepotrebujem

avsak jules ma pre mna vyssiu autoritu pretoze ako jediny so mnou aktivne riesil onboarding do kodu

kludne anglicke readme je vpoho podla mna, aj ked nevidim ziadnu pridanu hodnotu v tom, a hlavne by som to nebral ako blocker, ale ako refaktoring a konzistentnost (imho ale fakt to neprida hodnotu, ale tak tiez mam OCD tak kludne niekomu spravim radost ak tam nie je nic co by ho skrelo viac)